build/dat.gui.js

```diff
@@ -1285,255 +1285,255 @@ var FunctionController = function (_Controller) {
 }(Controller);
 
 var ColorController = function (_Controller) {
-  inherits(ColorController, _Controller);
```

Comment:
Seems my build toolchain put 4 spaces in here vs 2, i'll look into making this diff less bad,

Reply:
In the meantime, GH can ignore whitespace in the diff: https://github.com/dataarts/dat.gui/pull/184/files?w=1

Reply:
This is quite odd, as its not consistently doing 4 spaces, the rest of the file is 2. I tried explicitly specifying 2 spaces and it didn't fix it. Actually made the diff worse. I think it's not worth trying to figure out what rollup / babel madness is causing this as it's generated code. Are you ok with this indentation inconsistency here?

Reply:
Let's just remove the `build/*` files from the PR, can compile them at next release.